shared/templates/template_OVAL_timer_enabled

```diff
@@ -0,0 +1,47 @@
+<def-group>
+
+{{% if init_system == "systemd" and target_oval_version >= [5, 11] %}}
```

Comment:
Any special reason not to add support for OVAL 5.10?

Reply:
@jan-cerny you can probably reuse the implementation of service enabled for OVAL 5.10. See: https://github.com/ComplianceAsCode/content/blob/master/shared/templates/template_OVAL_service_enabled#L60

Reply:
@yuumasato I haven't looked into that yet.

@ggbecker That one probably not, because that's for SysV init and my understaning is that timers are a feature of SystemD. But something like https://github.com/ComplianceAsCode/content/blob/master/shared/templates/template_OVAL_service_disabled#L199  could work.